NuRadioMC/simulation/simulation.py

```diff
@@ -1757,7 +1759,7 @@ def add_filtered_noise_to_channels(self, evt, station, channel_ids):
 
     def _add_empty_channel(self, station, channel_id):
         """ Adds a channel with an empty trace (all zeros) to the station with the correct length and trace_start_time """
-        trigger = station.get_primary_trigger()
+        trigger = station.get_first_trigger()
```

Comment:
I understand that in principle the primary trigger matters. But I believe the idea was that we can simulate "identical" triggers with different thresholds and we keep the freedom to specify the trigger we are most interested in. Correct @cg-laser? Otherwise I agree that in principle first is correct here.